src/Microsoft.ML/CSharpApi.cs

```diff
@@ -10805,23 +10805,23 @@ public ColumnCopier(params (string inputColumn, string outputColumn)[] inputOutp
             
             public void AddColumn(string inputColumn)
             {
-                var list = Column == null ? new List<Microsoft.ML.Transforms.CopyColumnsTransformColumn>() : new List<Microsoft.ML.Transforms.CopyColumnsTransformColumn>(Column);
-                list.Add(OneToOneColumn<Microsoft.ML.Transforms.CopyColumnsTransformColumn>.Create(inputColumn));
+                var list = Column == null ? new List<Microsoft.ML.Transforms.CopyColumnsTransformerColumn>() : new List<Microsoft.ML.Transforms.CopyColumnsTransformerColumn>(Column);
+                list.Add(OneToOneColumn<Microsoft.ML.Transforms.CopyColumnsTransformerColumn>.Create(inputColumn));
```

Comment:
>CopyColumnsTransformerColumn [](start = 64, length = 28)

This is a little unfortunate, maybe there is a way we can avoid this rename.

Reply:
I believe you've solved this based on a rename

---
In reply to: [212749809](https://github.com/dotnet/machinelearning/pull/706#discussion_r212749809) [](ancestors = 212749809)